content/projects/jcasc/dev-tools.adoc

```diff
@@ -0,0 +1,61 @@
+---
+layout: gsocproject
+title: "Dev-Tools for JCasC"
+goal: "IDE integrations for JCasC, along with a better testing architecture for the JSON schema, and addition of configuration extension points"
+category: Plugin
+year: 2019
+status: Active
+showGoogleDoc: false
+student: sladyn98
+mentors:
+- "timja"
+- "casz"
+links:
+  gitter: https://gitter.im/jenkinsci/jcasc-devtools-project
+  draft: https://docs.google.com/document/d/1aPfkmyMQRCcipVa0htFt-i7X_pkQHSVKfR5qT60HQZ8/edit?usp=sharing
+opengraph:
+  image: /images/logos/JCasC/JCasC.png
+---
+
+JCasC Development tools 
```

Comment:
Usually the system wants a level 3 heading to be preceded by a level 2 heading.  Are you intentionally making this text standard size rather than making it a heading?
```suggestion
== JCasC Development tools 
```

Reply:
I just followed the Google template. I will make this change though :+1